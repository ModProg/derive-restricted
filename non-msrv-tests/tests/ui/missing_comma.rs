#[derive_where::derive_where(Clone Debug; T)]
struct Test1<T>(T);

#[derive_where::derive_where(Clone; A B)]
struct Test2<A, B>(A, B);

fn main() {}
