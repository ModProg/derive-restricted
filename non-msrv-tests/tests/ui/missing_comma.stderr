error: expected `;` or `,
 --> tests/ui/missing_comma.rs:2:22
  |
2 | #[derive_where(Clone Debug; T)]
  |                      ^^^^^

error: expected `,`
 --> tests/ui/missing_comma.rs:6:25
  |
6 | #[derive_where(Clone; A B)]
  |                         ^
